modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -47,6 +51,13 @@ def verify_power_of_attorney
           verifier.verify(header('X-Consumer-PoA'))
```

Comment:
Should we be getting rid of the PoA header all together?

Reply:
I will remove that and change the verifier on this task https://github.com/department-of-veterans-affairs/vets-contrib/issues/1536